### DISCUSSION TITLE
[AnalyticalTable]: Missing attribute?

### DISCUSSION THREAD

**Discussion by reqyou** [UI5/webcomponents-react#5624] [Q&A]
Hey, 
is it possible that `expanded` is missing in the controlled state from AnalyticalTable?

If I check your `AnalyticalTableState` definition I see `expanded`, if I check `state` in `useControlledState` it seems also that `expanded` is applied, but its not applied to the component?

I would like to set `expanded` to `true` by default, so that all grouped columns are expanded like mentioned in the tanStack doc [https://tanstack.com/table/v8/docs/api/features/expanding](https://tanstack.com/table/v8/docs/api/features/expanding), which is also working nicely when I use directly the tan stack implementation. I would like to avoid calling `toggleAllRowsExpanded` on each update, to make sure that everything is expanded.

```
reactTableOptions={{
        initialState: {
          sortBy: sortingState,
          groupBy: ['myColumn'],
          expanded: true
        }}}
```

Best, 
reQ

**Comment by Lukas742** [comment]
Hi @reqyou 

the `AnalyticalTable` is using v7 of `react-table`, but the documentation you're referring to is for v8. You can find the v7 documentation [here](https://github.com/TanStack/table/blob/v7/docs/src/pages/docs/api/useExpanded.md).
